roles/common/tasks/python.yml

```diff
@@ -86,13 +86,16 @@
 - block:
   - name: uninstall python-setuptools
     package: name=python-setuptools state=absent
-    when: common.setuptools_version != "system"
+    when:
+      - common.setuptools_version != "system"
+      - ursula_os == 'ubuntu'
```

Comment:
Yum package manager uninstalls pip if we remove setuptool (does not happen on ubuntu). On Ubuntu if we pip install (in next task) the version we want it replaces it the system one.